src/main/scala/chisel3/experimental/util/algorithm/Bitwise.scala

```diff
@@ -0,0 +1,29 @@
+package chisel3.experimental.util.algorithm
+
+import chisel3._
+
+/** Map each bits to logical or of itself and all bits less siginificant than it.
+  * @example {{{
+  * LSBOr("b00001000".U) // Returns "b11111000".U
+  * LSBOr("b00010100".U) // Returns "b11111100".U
+  * LSBOr("b00000000".U) // Returns "b00000000".U
+  * }}}
+  */
```

Comment:
Good doc,
I remember I had an issue chipsalliance/rocket-chip#2616 years ago discussing this, this circuit seems to be high fan-out, but it can be optimized by the synthesis tool. Maybe adding a note here to users for the fanout issue?
```suggestion
/** Map each bits to logical or of itself and all bits less siginificant than it.
  * @example {{{
  * LSBOr("b00001000".U) // Returns "b11111000".U
  * LSBOr("b00010100".U) // Returns "b11111100".U
  * LSBOr("b00000000".U) // Returns "b00000000".U
  * }}}
  * This circuit seems to be high fan out, but synthesis tool should handle this.
  */
```